src/components/Comments/WriteComment.js

```diff
@@ -85,11 +84,13 @@ const WriteComment = ({
           {username}
         </Text>
         <View style={styles.anonimityContainer}>
-          <ToggleSwitchAnon
+          <ToggleSwitch
             value={isAnonimity}
             onValueChange={toggleSwitch}
             labelLeft="Anonymity"
-            labelColor={COLORS.gray8}
+            backgroundActive={colors.lightgrey}
+            backgroundInactive={colors.lightgrey}
+            styleLabelLeft={styles.switch}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ToggleSwitch` component is now using `colors.lightgrey` for both active and inactive backgrounds, which might not provide a clear visual indication of its state. Consider using different colors for active and inactive states.
```diff
- labelColor={COLORS.gray8}
+ backgroundActive={colors.lightgrey}
+ backgroundInactive={colors.lightgrey}
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->